draft-irtf-cfrg-hpke.md

```diff
@@ -245,17 +245,19 @@ HPKE variants rely on the following primitives:
     of the private key `skR` is assured that the ephemeral shared
     key is known only to the holder of the private key corresponding
     to `pkS`
-  - Nenc: The length in bytes of an encapsulated key from this KEM
+  - Nzz: The length in bytes of a shared secret produced by this KEM
+  - Nenc: The length in bytes of an encapsulated key produced by this KEM
   - Npk: The length in bytes of an encoded public key for this KEM
 
-* A Key Derivation Function:
+* A Key Derivation Function (KDF):
   - Hash(m): Compute the cryptographic hash of input message `m`
```

Comment:
Can we get rid of `Hash()` now?

Reply:
I also had this thought but didn't follow it yet, thanks for bringing it up. I think we can get rid of Hash. I'll have a look where in the spec this would need adjustment.

Reply:
Removed Hash (and Nb) with commit https://github.com/cfrg/draft-irtf-cfrg-hpke/pull/50/commits/877237816152d2a58dc866bba1c9e12b3c4a5a93.